zeebe-cluster/values.yaml

```diff
@@ -25,6 +25,7 @@ logLevel: info
 
 gateway:
   replicas: 1
+  podLabels: {}
```

Comment:
@caugustus-ezcater can we avoid having these extra empty default values in the `values.yaml` file?

Reply:
@salaboy - removed the default values. I like to add defaults to serve as documentation, so that users don't have to inspect the templates themselves, but it does fill up the file.

Reply:
@caugustus-ezcater i tend to agree.. it is hard to choose between those two options.. we should copy other communities .. thanks for removing those defaults now.. I will research a bit more and then maybe add them again if I find that other popular charts are doing that